Microsoft.Toolkit.Uwp.UI/Extensions/FrameworkElement/FrameworkElementExtensions.LogicalTree.cs

```diff
@@ -179,6 +179,23 @@ public static partial class FrameworkElementExtensions
                     goto Start;
                 }
             }
+            else if (element is ContentPresenter contentPresenter)
+            {
+                // Sometimes ContentPresenters are used in control templates instead of ContentControls
+                // Therefore we should still check if it's Content is a FrameworkElement and match.
```

Comment:
Minor nit 😄
```suggestion
                // Sometimes ContentPresenter is used in control templates instead of ContentControl,
                // therefore we should still check if its Content is a matching FrameworkElement instance.
```